core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.directive.html

```diff
@@ -50,6 +50,9 @@
       <p ng-if="triedToCopyText">
         Paragraphs are not allowed to be copied. Please make sure to translate the text.
       </p>
+      <p ng-if="incompleteTranslationError">
+        Translation is incomplete.
```

Comment:
Let's also tell the user how to correct the issue.

```suggestion
        Translation is incomplete. Please make sure all elements in the original content are included in the translated content.
```

Reply:
Done